package.json

```diff
@@ -4,9 +4,12 @@
   "description": "Link shortener for Singapore government.",
   "main": "src/server/index.js",
   "scripts": {
-    "lint-js": "eslint --ext .js,.jsx --no-eslintrc -c .eslintrc.json --ignore-path .gitignore --fix .",
-    "lint-ts": "eslint --ext .ts --no-eslintrc -c .eslintrc.ts.json --ignore-path .gitignore --fix .",
+    "lint-js": "eslint --ext .js,.jsx --no-eslintrc -c .eslintrc.json --ignore-path .gitignore .",
+    "lint-ts": "eslint --ext .ts --no-eslintrc -c .eslintrc.ts.json --ignore-path .gitignore .",
+    "lint-js:fix": "eslint --ext .js,.jsx --no-eslintrc -c .eslintrc.json --ignore-path .gitignore --fix .",
+    "lint-ts:fix": "eslint --ext .ts --no-eslintrc -c .eslintrc.ts.json --ignore-path .gitignore --fix .",
     "lint": "npm run lint-js && npm run lint-ts",
+    "lint:fix": "npm run lint-js:fix && npm run lint-ts:fix",
```

Comment:
Noticed the naming used for linting with auto-fix. I believe the naming convention used for these operations is `format`.

Reply:
`lint` is fine

Reply:
how about `lint` for fixing and `lint:ci` for CI?

Reply:
I'm concerned it might not be immediately obvious what `lint:ci` does. I think that having `lint` and `lint:fix` would make it more obvious.

Reply:
OK, just trying to save some keystrokes here.